numpy/core/tests/test_indexing.py

```diff
@@ -141,10 +143,10 @@ def test_empty_fancy_index(self):
         assert_raises(IndexError, a.__getitem__, b)
 
     def test_ellipsis_index(self):
-        # Ellipsis index does not create a view
         a = np.array([[1, 2, 3],
                       [4, 5, 6],
                       [7, 8, 9]])
+        assert_is_not(a[...], a)
```

Comment:
I'd prefer a plain old assert here. At some point we are going to drop nose, it is abandonware, and I'd like to keep the dependencies to a minimum an in one spot.